src/content/zh-tw/fundamentals/payments/index.md

```diff
@@ -0,0 +1,73 @@
+project_path：/web/fundamentals/_project.yaml book_path：/web/fundamentals/_book.yaml
```

Comment:
book_path：/web/fundamentals/_book.yaml
should in a new line

Reply:
Yes, these need to be on separate lines.